src/actors/tornado_scanner/tornado_scanner.py

```diff
@@ -0,0 +1,91 @@
+import json
+import os
+import re
+import sys
+import ast
+
+
+def is_tornado_package(main_file_path):
+    with open(main_file_path, 'r') as f:
+        tree = ast.parse(f.read())
+        for node in ast.walk(tree):
+            if isinstance(node, (ast.Import, ast.ImportFrom)):
+                for elem in node.names:
+                    if 'tornado' in elem.name:
+                        return True
```

Comment:
If you want to decrease one level of indentation:
```
(...)
if not isinstance(node, (ast.Import, ast.ImportFrom)): 
    continue
for elem in node.names:
   (...)
```

Reply:
done